src/main/steps/dss-update/check-your-answers/content.test.ts

```diff
@@ -5,19 +5,17 @@ import { CommonContent, generatePageContent } from '../../common/common.content'
 import { generateContent } from './content';
 
 const enContent = {
-  serviceName: 'DSS Update Case',
-  title: 'Check your Answers',
-  change: 'Edit',
-  continue: 'Accept and continue',
-  statementOfTruth: 'I beleive that the facts stated in this case are true.',
+  serviceName: 'Update an appeal to the First-tier Tribunal',
+  title: 'Check answers before submitting your update',
+  change: 'Change',
+  continue: 'Accept and send',
+  statementOfTruth: 'By updating this case you are confirming that, to the best of your knowledge, the details you are providing are correct.',
   submitApplicationText: 'Now submit your Application',
-  listOfDocuments: 'List of documents uploaded',
   errorSummaryMessage: 'There is a problem',
-  declarationText:
-    'I understand that proceedings for contempt of court may be brought against anyone who makes, or causes to be made, a false statement in a document verified by a statement of truth without an honest beleif in its truth.',
   keys: {
-    fileName: 'File name',
-    description: 'Description',
+    information: 'Additional information',
+    fileName: 'Additional documents',
```

Comment:
Better to change the left hand side name to match with the right hand side description?
Example: fileName: 'Additional documents' can be changed to 
documents: 'Additional documents' or something